zbus/tests/e2e.rs

```diff
@@ -773,6 +772,7 @@ async fn iface_and_proxy_(p2p: bool) {
     debug!("Client connection created: {:?}", client_conn);
     debug!("Service connection created: {:?}", service_conn);
 
+    let listen = event.listen();
```

Comment:
Why this change? Is it relevant to the update? If not, please split into separate commit with its own log/justification.

Reply:
I was reverting a part of the first commit, but it ended up in the wrong commit